lib/internal/url.js

```diff
@@ -874,32 +871,32 @@ class URLSearchParams {
     this[context] = null;
   }
 
-  [util.inspect.custom](recurseTimes, ctx) {
+  [util.inspect.custom](depth, opts) {
     if (!this || !this[searchParams] || this[searchParams][searchParams]) {
       throw new TypeError('Value of `this` is not a URLSearchParams');
     }
 
-    if (typeof recurseTimes === 'number' && recurseTimes < 0)
-      return ctx.stylize('[Object]', 'special');
+    if (typeof depth === 'number' && depth < 0)
+      return opts.stylize('[Object]', 'special');
 
-    const separator = ', ';
-    const innerOpts = Object.assign({}, ctx);
-    if (recurseTimes !== null) {
-      innerOpts.depth = recurseTimes - 1;
+    var separator = ', ';
```

Comment:
Ideally, I agree - but since it was missed the first time around in https://github.com/mscdex/io.js/commit/6b6123cfbe90141464bb86400a5f26c835c50f37#diff-7fa9474fccb31c3ccef5d728fbd5248bR865 and I don't think performance regressions are run against every small refactoring I think a comment would still be useful in this particular case. 

If we can figure how to do it in an automated way that would be great, I don't think contributors are not very familiar with the basic performance tooling like `--trade-deopt` let alone can understand their output very easily.